ui/package.json

```diff
@@ -28,7 +28,7 @@
     "@types/react-router-dom": "^4.2.3",
     "@types/superagent": "^3.5.7",
     "ansi-to-react": "^6.1.6",
-    "argo-ui": "git+https://github.com/argoproj/argo-ui.git",
+    "argo-ui": "git+https://github.com/alexmt/argo-ui.git#customizable-page-breadcrumbs",
```

Comment:
Hi Alex, I thought only the lock file was needed to be changed to pick up the changes in argo-ui

Reply:
I had to temporary change package.json to use my fork with un-merged changes. Reverted changes.